timeline.html

```diff
@@ -1 +1,395 @@
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-md-1">
+                        <a href="#">Like</a>
+                      </div>
+                      <div class="col-md-10 col-md-offset-1">
+                        Three other people like this.
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="row comment">
+                  <div class="col-md-2">
+                      <img src="img/user_silhouette_generic.gif" class="img-responsive">
+                  </div>
+                  <div class="col-md-10">
+                    <div class="row">
+                      <div class="col-md-12">
+                        <a href="#">Harry Potter</a> <span class="post-date">said on Friday 5/22/14:</span>
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-md-12">
+                        <p>Mudbloods yew pumpkin juice phials Ravenclaw’s Diadem 10 galleons Thieves Downfall. Ministry-of-Magic mimubulus mimbletonia Pigwidgeon knut phoenix feather other minister Azkaban. Hedwig Daily Prophet treacle tart full-moon Ollivanders You-Know-Who cursed. Fawkes maze raw-steak Voldemort Goblin Wars snitch Forbidden forest grindylows wool socks..</p>
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-md-1">
+                        <a href="#">Like</a>
+                      </div>
+                      <div class="col-md-7 col-md-offset-1">
+                        Three other people like this.
+                      </div>
+                      <div class="col-md-1">
+                        <a href="#">Delete</a>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="row">
+                  <div class="col-md-12 comment-entry">
+                    <div class="row">
+                      <div class="col-md-12">
+                        <textarea rows="6" placeholder="Write a comment..."></textarea>
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-md-2">
+                        <a href="#">Cancel</a>
+                      </div>
+                      <div class="col-md-2 col-md-offset-8">
+                        <button>Comment</button>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </article>
+        </div>
+      </div>
```

Comment:
I can tell you guys spent a lot of time on this file. The HTML on this is very good.